test/Crypto/Secp256k1Prop.hs

```diff
@@ -355,5 +368,39 @@ prop_eqImportImpliesEqSecKey = property $ do
     k0 === k1
 
 
+prop_schnorrSignatureParseInvertsSerialize :: Property
+prop_schnorrSignatureParseInvertsSerialize = property $ do
+    sk <- forAll secKeyGen
+    msg <- forAll $ bytes (singleton 32)
+    let kp = keyPairCreate sk
+    sig <- maybe failure pure $ schnorrSign kp msg
+    let serialized = exportSchnorrSignature sig
+    annotateShow serialized
+    annotateShow (BS.length serialized)
+    let parsed = fromJust $ importSchnorrSignature serialized
+    parsed === sig
```

Comment:
Rather than using `fromJust` here just make the assertion `parsed === Just sig`. I think this will make failures a little easier to diagnose. `fromJust` has one of the worst error messages IIRC.